docs/docs/services/qix-engine/apis/qix/introduction.md

```diff
@@ -31,6 +29,22 @@ Header | Description
 `X-Qlik-Session: <guid>` | Defines the session id, regardless of the user. If you use this header, the websocket URL could simply be `ws://hostname:9076/app/`.
 `Authorization: Bearer <token>` | If the [JWT feature](../../../../tutorials/authorization.md) is enabled, this header will identify the user and its attributes.
 
+### Session Sharing
+
+Qlik Associative Engine uses the concept of _sessions_ to identify the state of connections to the QIX API. When making
+a new connection either a new unique session is created, or an exsting session is shared. This makes it possible to
```

Comment:
`or an exsting session is shared` -> `or the connection gets attached to an existing session`?

Reply:
:+1: yes, better.